config/sync/field.field.media.document.field_media_file.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 90daccc1-1a52-4849-bb69-2130cc7f3dc0
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_file
+    - media.type.document
+  module:
+    - file
+id: media.document.field_media_file
+field_name: field_media_file
+entity_type: media
+bundle: document
+label: File
+description: ''
+required: true
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:file'
+  handler_settings: {  }
+  file_directory: '[date:custom:Y]-[date:custom:m]'
+  file_extensions: pdf
```

Comment:
Consider talking to the business about what types of files they typically upload.

Limiting the allowed file extensions to PDF seems a bit too restrictive to me.